fraud/service.go

```diff
@@ -1,57 +1,99 @@
 package fraud
 
 import (
+	"bytes"
 	"context"
 	"encoding/hex"
 	"errors"
 	"fmt"
 	"sync"
-	"time"
 
 	"github.com/ipfs/go-datastore"
+	"github.com/libp2p/go-libp2p-core/host"
 	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/libp2p/go-libp2p-core/protocol"
 	pubsub "github.com/libp2p/go-libp2p-pubsub"
-)
 
-const (
-	// fetchHeaderTimeout duration of GetByHeight request to fetch an ExtendedHeader.
-	fetchHeaderTimeout = time.Minute * 2
+	"github.com/celestiaorg/celestia-node/params"
 )
 
-// service is responsible for validating and propagating Fraud Proofs.
+// fraudRequests is amount of external requests that will be done in order to get fraud proofs from another peers.
```

Comment:
Either "another peer" or "other peers"
```suggestion
// fraudRequests is the amount of external requests that will be tried to get fraud proofs from other peers.
```